DataFormats/HGCDigi/interface/PHGCSimAccumulator.h

```diff
@@ -0,0 +1,156 @@
+#ifndef DataFormats_HGCDigi_PHGCSimAccumulator_h
+#define DataFormats_HGCDigi_PHGCSimAccumulator_h
```

Comment:
This file should likely be located somewhere in `SimDataFormats` instead. None of the existing sub-packages were crystal-clear match, so I ask for better suggestions.